bluebrain/repo-bluebrain/packages/py-diameter-synthesis/package.py

```diff
@@ -22,10 +23,15 @@ class PyDiameterSynthesis(PythonPackage):
     depends_on("py-setuptools-scm", type="build")
 
     depends_on("py-click@7.0:", type=("build", "run"))
-    depends_on("py-numpy@1.15.0:", type=("build", "run"))
-    depends_on("py-scipy@0.13.3:", type=("build", "run"))
-    depends_on("py-matplotlib@2.2:", type=("build", "run"))
-    depends_on("py-pandas@0.24:", type=("build", "run"))
+    depends_on("py-numpy@1.33.0:", type=("build", "run"), when="@0.5:")
```

Comment:
```suggestion
    depends_on("py-numpy@1.22.0:", type=("build", "run"), when="@0.5:")
```
latest release is 1.25

Reply:
Indeed, thanks! :smile: